externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/util/KyuubiJdbcUtil.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.jdbc.util
+
+import java.sql.{Connection, Statement}
+
+import scala.util.{Failure, Success, Try}
+
+import org.apache.kyuubi.{KyuubiSQLException, Logging}
+import org.apache.kyuubi.config.KyuubiConf
+import org.apache.kyuubi.engine.jdbc.connection.ConnectionProvider
+import org.apache.kyuubi.engine.jdbc.dialect.{JdbcDialect, JdbcDialects}
+
+object KyuubiJdbcUtil extends Logging {
+
+    def initializeJdbcSession(kyuubiConf: KyuubiConf): Unit = {
```

Comment:
- if a method is used in only one place, creating a `Util` object is not a good practice.

- Kyuubi uses `AbcUtils` style, not `AbcUtil`.

- `org.apache.kyuubi.util.JdbcUtils#withCloseable` can simplify code

Reply:
> * if a method is used in only one place, creating a `Util` object is not a good practice.
> * Kyuubi uses `AbcUtils` style, not `AbcUtil`.
> * `org.apache.kyuubi.util.JdbcUtils#withCloseable` can simplify code
* emm, I am considering whether to add a method to execute Session Init SQL in Util
* I refer to the **KyuubiSparkUtil** format before, I will update later
* I'll look at the **JdbcUtils** code in more detail later